Sources/App/Controllers/API/API+PackageController+GetRoute+Model.swift

```diff
@@ -197,7 +182,7 @@ extension API.PackageController.GetRoute {
                 preReleaseReference: result.preReleaseVersion?.reference,
                 fundingLinks: result.repository.fundingLinks,
                 swift6Readiness: swift6Readiness,
-                forkedFromResult: forkedFromResult
+                forkedFromInfo: forkedFromInfo
```

Comment:
The model `init` now doesn't do any logic with `ForkedFromInfo` - it just assigns it. That makes the tests simpler, see below.